### DISCUSSION TITLE
How can I add container to a root of the project?

### DISCUSSION THREAD

**Discussion by abhinav-anshul** [zeit-ui/react#702] [General]
Hello all,

I would like to wrap my `index.tsx` file in a container, so it just doesn't stretch beyond a certain screen size(useful on large monitors). I understand this can be done easily using a maxWidth style on the root div, but I was hoping to achieve the same using the Container/Page component, similar to how the material UI/ant-d library does.

I have tried wrapping it in a Page component but it doesn't work as expected.

is there anything obvious that I am missing?

Thanks so much for reading!

**Comment by unix** [comment]
`Page` has a default width (`100% - 100pt`), which you can change [width](https://geist-ui.dev/en-us/guide/scaleable) to constrain the page:

```tsx
<Page width="800px">
  <App />
</Page>
```
